packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -22,7 +21,7 @@ import 'shortcuts.dart';
 // happens to be of the correct type. getParent should be O(1), since we
 // always return false at the first ancestor.
 BuildContext _getParent(BuildContext context) {
-  BuildContext parent;
+  late BuildContext parent;
```

Comment:
might be worth making this `final` as a further check that the callback only runs once

Reply:
Done